src/tests/actions/blogActions.test.js

```diff
@@ -86,6 +85,23 @@ describe('blogActions', () => {
       const dispatchedActions = store.getActions();
       expect(dispatchedActions[0].type).toBe('ADD_NEW_ENTRY');
     });
+
+    test('checks if action payload is correct and there is uuid', () => {
```

Comment:
Having an "and" in the test name is usually a code smell that the test is testing too much. However, in this case, the phrase "action payload is correct" implies the fact that there is an uuid. So, just drop the final part.